src/org/usfirst/frc4904/sovereignty/FusedSensor.java

```diff
@@ -0,0 +1,86 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.InvalidSensorException;
+
+public class FusedSensor<T> implements Fusible<T> {
+	protected final Fusible<T> primarySensor;
+	protected final Fusible<T> fallbackSensor;
+	protected final Fusible<T> lastResortSensor;
+
+	public FusedSensor(T defaultValue, Fusible<T> primarySensor, Fusible<T> secondarySensor, Fusible<T>... sensors) {
+		Fusible<T> lastSensor = sensors[sensors.length - 1];
+		for (int i = sensors.length - 2; i >= 0; i--) {
+			lastSensor = new FusedSensor<>(sensors[i], lastSensor);
+		}
+		lastSensor = new FusedSensor<>(secondarySensor, lastSensor);
+		this.primarySensor = primarySensor;
+		this.fallbackSensor = lastSensor;
+		this.lastResortSensor = new Fusible<T>() {
+			@Override
+			public T getValue() {
+				return defaultValue;
+			}
+
+			@Override
+			public boolean trustable() {
+				return false;
```

Comment:
@c0d3rman well, you kinda hit on either side of the target. It's really a mixture of both of those. The status of the sensor, i.e. communicating, isn't enough. You also need to know if the data is okay for use, which might not be the case despite having comms for a plethora of reasons. The sensor can most likely determine for itself how accurate the data is. If it is, say, roughly normal it would probably return `true`. So `trustable` is really the confidence the sensor has in itself. 

In this particular case with the lastResortSensor, I would argue that the application of that confidence or perhaps the scope of that confidence is nuanced. In the FusedSensor, lastResortSensor is only used when both the primary and fallback sensor are un-trustable. At this point, it makes sense to deem any further value passed from the FusedSensor un-trustable, as they are not from any of the sensors provided. The last resort sensor is simply a courtesy to the user intended to allow them to handle the failing of both sensors in a more elegant way than catching a null pointer. It is not, however, meant to actually be acted upon ever, and I would thus deem that value un-trustable.